ansible/tasks/test-image.yml

```diff
@@ -13,10 +27,21 @@
   failed_when: retval.rc != 0
   when: ebssurrogate_mode
 
+- name: Re-enable PG Sodium references in config
+  become: yes
+  become_user: postgres
+  shell:
+    cmd: mv /etc/postgresql/postgresql.conf.bak /etc/postgresql/postgresql.conf
```

Comment:
seems out of sync with the use of sed to disable above?

Reply:
I had to temporarily remove sodium related entry `pgsodium.getkey_script` from the config file to start the process. So did `sed` to backup the original config and after running pgtap, now revert to the original config.  

Before this PR pgtap used to run against config from /var/lib/postgresql, now with this PR pgtap uses /etc/postgresql/postgresql.conf to run the tests.